docs/contributors/patching/maintain-ubuntu-desktop-software.md

```diff
@@ -0,0 +1,1110 @@
+---
+relatedlinks: "[PackagingWithGit &#32; - &#32; Debian Wiki](https://wiki.debian.org/PackagingWithGit)"
+---
+
+(maintain-ubuntu-desktop-software)=
+# Maintain Ubuntu Desktop software
+
+These are some common, day-to-day operations to build, maintain and package GNOME software for Ubuntu Desktop using the Git Build Package (`gbp`) workflow.
+
+We'll use the `gnome-control-center` repository as an example.
+
+:::{note}
+Some applications on Ubuntu Desktop are developed outside of Salsa and GNOME. They have their own, separate workflows, which aren't described in this guide. The Desktop Team can help you find the right instructions on Matrix: {matrix}`desktop-dev`.
+:::
+
+
+## Prerequisites
+
+Before you start, follow the instructions in {ref}`set-up-git-for-ubuntu-desktop-work`.
+
+
+## Team members, contributors and permissions
+
+If you're a community contributor and not a member of the Ubuntu Desktop Team, you have to send your contributions through a merge request:
+
+1. Fork the Git repository on Salsa.
+1. Follow this guide and make the changes in your fork.
+1. Open a Salsa merge request from your fork to the original repository.
+
+In some projects, no Ubuntu branch has been created in a long time. You might have to ask the Desktop Team to create the new branch for you. You can contact them on Matrix at {matrix}`desktop-dev` or [on the Canonical Mattermost](https://chat.canonical.com/canonical/channels/desktop).
+
+
+## Local changes
+
+With the suggested Git configuration, any non-committed changes (file modifications, additions or removal) halt the build because they won't be included. This is a safety reminder.
```

Comment:
```suggestion
With the suggested Git configuration, any non-committed changes (file modifications, additions, or removal) halt the build because they won't be included. This is a safety reminder.
```